src/objects/setup_configuration/models/objecttypes.py

```diff
@@ -0,0 +1,17 @@
+from django_setup_configuration.fields import DjangoModelRef
```

Comment:
Both `models/` and `steps/` are missing `__init__.py`, not sure if it causes errors, but I do see warnings about it: 

```
WARNINGS:
?: (utils.W001) Directory %s does not contain an `__init__.py` file
        HINT: Consider adding this module to make sure tests are picked up
?: (utils.W001) Directory %s does not contain an `__init__.py` file
        HINT: Consider adding this module to make sure tests are picked up
```